packages/ui/core-components/src/lib/organisms/layout/BreadCrumbs.svelte

```diff
@@ -55,7 +55,7 @@
 	$: crumbs = buildCrumbs(pathArray);
 </script>
 
-<div class="flex items-start mt-8 sm:mt-12 whitespace-nowrap overflow-auto">
+<div class="flex items-start mt-[6px] whitespace-nowrap overflow-auto">
```

Comment:
This is a regression, we currently reduce whitespace between the breadcrumbs and the header on mobile. 

```suggestion
<div class="flex items-start whitespace-nowrap overflow-auto">
```